terraform/examples/io500/run_io500-sc22.sh

```diff
@@ -39,93 +39,71 @@ SERVER_LIST=$(awk -vORS=, '{ print $1 }' "${SCRIPT_DIR}/hosts_servers" | sed 's/
 
 # Source config file to load variables
 if [[ -f "${CONFIG_FILE}" ]]; then
+  # shellcheck source=/dev/null
   source "${CONFIG_FILE}"
 fi
 
-# Set environment variable defaults if not already set
-# This allows for the variables to be set to different values externally.
 IO500_STONEWALL_TIME="${IO500_STONEWALL_TIME:-5}"
-IO500_INSTALL_DIR="${IO500_INSTALL_DIR:-/usr/local}"
-IO500_DIR="${IO500_DIR:-${IO500_INSTALL_DIR}/${IO500_VERSION_TAG}}"
-IO500_RESULTS_DFUSE_DIR="${IO500_RESULTS_DFUSE_DIR:-${HOME}/daos_fuse/${IO500_VERSION_TAG}/results}"
-IO500_RESULTS_DIR="${IO500_RESULTS_DIR:-${HOME}/${IO500_VERSION_TAG}/results}"
-DAOS_POOL_LABEL="${DAOS_POOL_LABEL:-io500_pool}"
-DAOS_CONT_LABEL="${DAOS_CONT_LABEL:-io500_cont}"
+IO500_DIR="${IO500_DIR:-"/opt/${IO500_VERSION_TAG}"}"
 
-log() {
-  local msg
-  local print_lines
-  msg="$1"
-  print_lines="$2"
-  # shellcheck disable=SC2155,SC2183
-  local line=$(printf "%80s" | tr " " "-")
-  if [[ -t 1 ]]; then tput setaf 14; fi
-  if [[ "${print_lines}" == 1 ]]; then
-    printf -- "\n%s\n %-78s \n%s\n" "${line}" "${msg}" "${line}"
-  else
-    printf -- "\n%s\n\n" "${msg}"
-  fi
-  if [[ -t 1 ]]; then tput sgr0; fi
-}
+IO500_DFUSE_DIR="${IO500_DFUSE_DIR:-"${HOME}/daos_fuse/${IO500_VERSION_TAG}"}"
+IO500_DATAFILES_DFUSE_DIR="${IO500_DATAFILES_DFUSE_DIR:-"${IO500_DFUSE_DIR}/datafiles"}"
+IO500_RESULTS_DFUSE_DIR="${IO500_RESULTS_DFUSE_DIR:-"${IO500_DFUSE_DIR}/results"}"
 
-log_error() {
-  # shellcheck disable=SC2155,SC2183
-  if [[ -t 1 ]]; then tput setaf 160; fi
-  printf -- "\n%s\n\n" "${1}" >&2;
-  if [[ -t 1 ]]; then tput sgr0; fi
-}
+IO500_RESULTS_DIR="${IO500_RESULTS_DIR:-"${HOME}/${IO500_VERSION_TAG}/results"}"
 
-log_section() {
-  log "$1" "1"
-}
+DAOS_POOL_LABEL="${DAOS_POOL_LABEL:-io500_pool}"
+DAOS_CONT_LABEL="${DAOS_CONT_LABEL:-io500_cont}"
 
 fix_admin_cert_permissions() {
   # In order to run dmg you must run it with a key which is owned by you
   # This is a hack to allow daos-user to run dmg
+  log.debug "BEGIN: fix_admin_cert_permissions()"
   if [[ -f /etc/daos/certs/admin.key ]];then
     clush --hostfile="${SCRIPT_DIR}/hosts_clients" --dsh sudo chown "${SSH_USER}":"${SSH_USER}" /etc/daos/certs/admin*
   else
-    log_error "Unable to fix admin cert permissions. admin.key does not exist."
+    log.error "Unable to fix admin cert permissions. admin.key does not exist."
   fi
+  log.debug "END: fix_admin_cert_permissions()"
 }
 
 unmount_defuse() {
-  if [[ -d "${IO500_RESULTS_DFUSE_DIR}" ]]; then
-    log_section "Unmount DFuse mountpoint ${IO500_RESULTS_DFUSE_DIR}"
+  log.info "Attempting to unmount DFuse mountpoint ${IO500_DFUSE_DIR}"
+  if [[ -d "${IO500_DFUSE_DIR}" ]]; then
```

Comment:
nit, `findmnt` could be more adapted to check if a mountpoint is mounted or not

Reply:
Fixed